packages/lib/gpcircuits/test/proto-pod-gpc.spec.ts

```diff
@@ -20,393 +25,391 @@ import {
   testArray2Bits
 } from "./common";
 
-describe("proto-pod-gpc.ProtoPODGPC should work", function () {
-  // Circuit compilation sometimes takes more than the default timeout of 2s.
-  let circuit: WitnessTester<
-    ProtoPODGPCInputNamesType,
-    ProtoPODGPCOutputNamesType
-  >;
-  const MAX_OBJECTS = 3;
-  const MAX_ENTRIES = 10;
-  const MERKLE_MAX_DEPTH = 8;
-
-  const sampleInput: ProtoPODGPCInputs = {
-    // Object modules [MAX_OBJECTS].
-    objectContentID: [
-      21748523748810072846647845097417136490972606253431724953054174411568740252986n,
-      8121973595251725959527136190050016648811901981184487048534858036206640503232n,
-      21748523748810072846647845097417136490972606253431724953054174411568740252986n
-    ],
-    /*PUB*/ objectSignerPubkeyAx: [
-      13277427435165878497778222415993513565335242147425444199013288855685581939618n,
-      13277427435165878497778222415993513565335242147425444199013288855685581939618n,
-      13277427435165878497778222415993513565335242147425444199013288855685581939618n
+const MAX_OBJECTS = 3;
+const MAX_ENTRIES = 10;
+const MERKLE_MAX_DEPTH = 8;
+
+const sampleInput: ProtoPODGPCInputs = {
```

Comment:
low priority suggestion: may be useful to include a reference to the code that generates these sample inputs